src/building-blocks/FormFieldSet.tsx

```diff
@@ -39,7 +39,8 @@ class FormFieldSet extends Component<IFormFieldSetProps> {
     }
 
     const formModel = this.props.formManager.formModel;
-    return fieldConfigs.filter(fieldConfig => !filterFieldConfig(fieldConfig, { model: formModel }));
+    return fieldConfigs.filter(
+      fieldConfig => !filterFieldConfig(fieldConfig, { model: formModel, readOnly: true }));
```

Comment:
Was this supposed to have a `readOnly` before / why is this logic being added now?

Reply:
Yes, I believe this was an oversight. The filtered-out fieldConfigs would never be rendered due to being caught by the lower-level components, but in very rare cases like a FieldSet with 1 read-only fieldConfig, this would have caused an empty fieldset to render.

I'm gonna write a test for it I think.

Reply:
Yes this was an oversight. No, I can't test it yet, but will be able to in next PR.